ci/docker/runtime_functions.sh

```diff
@@ -755,6 +755,35 @@ build_ubuntu_gpu_cmake() {
     ninja -v
 }
 
+build_ubuntu_cpu_large_tensor() {
+    set -ex
+    build_ccache_wrappers
+    export CC="ccache gcc"
+    export CXX="ccache g++"
+    make  \
+        DEV=1                         \
+        ENABLE_TESTCOVERAGE=1         \
+        USE_CPP_PACKAGE=1             \
+        USE_INT64_TENSOR_SIZE=1       \
+        USE_SIGNAL_HANDLER=1          \
+        -j$(nproc)
+}
+
+build_ubuntu_gpu_large_tensor() {
+    set -ex
+    build_ccache_wrappers
+    make  \
```

Comment:
why not cmake?

Reply:
I saw both `make` and `cmake` used in runtime_functions.sh such as https://github.com/apache/incubator-mxnet/blob/master/ci/docker/runtime_functions.sh#L358

Sholud we only use `cmake` in the future?

Reply:
We have all agreed to use CMake, so we should not introduce new make builds unless there's a problem with CMake, this is my understanding.

Reply:
Sure, updated the process to cmake. I think we may need a seperate PR to update all runtime_functions to cmake otherwise it may cause confusion to future developers.